src/web/extractors.rs

```diff
@@ -285,7 +286,11 @@ impl FromRequest<ServerState> for BsoBody {
         }
         let mut config = JsonConfig::default();
         let max_request_size = req.state().limits.max_request_bytes as usize;
-        config.limit(max_request_size);
+        config.limit(max_request_size).content_type(|ct| {
+            ct.to_string() == "".to_owned()
+                || ct == mime::TEXT_PLAIN
```

Comment:
nit: I think we only need this line (at least according to test_storage.py). The mime crate is strongly typed and doesn't appear to parse an empty string content_type. Also JsonConfig always allows application/json

Reply:
Honestly, I was being super paranoid by that point. My test worked when I didn't include `mime::APPLICATION_JSON`, but I had no idea if that was a fluke or not.